CONTRIBUTING.md

```diff
@@ -66,9 +66,10 @@ Discussion about codebase improvements happens in GitHub issues and pull
 requests.
 
 The developer
-[mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
+[mailing list](https://groups.google.com/g/bitcoindev)
 should be used to discuss complicated or controversial consensus or P2P protocol changes before working on
 a patch set.
+Archives can be found in the [old mailing list](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/) maintained by the Linux Foundation.
```

Comment:
Probably best to use the new archive link throughout

Reply:
For links to the full archive and for the signup feature that seems fine.

But for links to specific threads (6 in total) I suggest we keep the old URL. At least for a few more years. They are redirected indefinitely, and not changing those links gives flexibly to switch to different archive later with or without generating a complete set of redirects.

Reply:
Also, looking at `https://gnusha.org/pi/bitcoindev/` the layout is not very pretty. So even for the old archive I would prefer sticking to the linuxfoundation URL. But is should clarify that the archive goes until December 2023.

Reply:
> Also, looking at `https://gnusha.org/pi/bitcoindev/` the layout is not very pretty. So even for the old archive I would prefer sticking to the linuxfoundation URL. But the text should clarify that this archive goes until December 2023.
> 
> (and I don't think Google itself should be recommended as the canonical archive)

I wasn't aware the Linux Foundation archive were not being updated anymore. It means that the new archive is the comprehensive source of truth in the present and probably is the best option.